### PR TITLE
Add Forever dependency

### PR DESCRIPTION
**Pull request by k-nut**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -14,6 +14,12 @@
 
 - [db-prices](https://github.com/juliuste/db-prices/) – "JavaScript client for the DB Sparpreise API."
 
+## Running
+```
+npm start
+# go to http://localhost:3009/
```
Comment (derhuerst) [inline readme.md]: I think `3009` is a rather unusual default port. What about changing it to `3000`?

Comment (juliuste) [reply readme.md]: Wouldn't `3000` be rather unusual as well? Compared to `8000` or `8080`

Comment (derhuerst) [reply readme.md]: `3000` is the common port for dev environments. `8000` and `8080` are often used as alternative ports for (production) HTTP on `80`, such as when adding a WebSocket server or backup service.

Comment (juliuste) [reply readme.md]: Yolo.

**Pull request merged**